src/iexamine.cpp

```diff
@@ -3484,6 +3485,7 @@ void iexamine::autodoc( player &p, const tripoint &examp )
 
     bool adjacent_couch = false;
     bool in_position = false;
+    bool deadened = false;
```

Comment:
Instead of "deadened", how about "needs_anesthesia".  It makes it a little clearer what the purpose is.